frontend/utility/AdvancedOutput.cpp

```diff
@@ -200,7 +200,19 @@ void AdvancedOutput::UpdateStreamSettings()
 		int keyint_sec = (int)obs_data_get_int(settings, "keyint_sec");
 		obs_service_apply_encoder_settings(main->GetService(), settings, nullptr);
 		if (!enforceBitrate) {
-			blog(LOG_INFO, "User is ignoring service bitrate limits.");
+			int MaxVBitrate, MaxABitrate;
```

Comment:
```suggestion
			int maxVideoBitrate;
			int maxAudioBitrate;
```

* Spell out words properly, there is no reason to abbreviate (or to abbreviate just specific parts). 
* Use explicit names for variables.
* Use `camelCase` for variables.
* Do not declare multiple variables on the same line, use separate line for each.

> [!NOTE]
> The abbreviation rule does _not_ apply to `max`, as it's an established term. For the same reason using `sin` is preferred, even though `sine` would be the more correct name for the mathematical function.